maintainers/maintainer-list.nix

```diff
@@ -78,6 +78,12 @@
       fingerprint = "2536 9E86 1AA5 9EB7 4C47  B138 6510 870A 77F4 9A99";
     }];
   };
+  _0xC45 = {
+    email = "jason@0xc45.com";
+    name = "Jason Vigil";
+    github = "0xC45";
+    githubId = 56617252;
+  };
```

Comment:
Perhaps add your `matrix`? Would be nice to have since it's `0xc45` instead of `0xc45` there.